src/components/SearchForm.js

```diff
@@ -171,56 +163,62 @@ const SearchForm = ({onFormChange}) => {
           <CharacterSelectButton
             name="characterRarity"
             label="UR"
-            onChange={setCharacterRarity}
-            currentSelect={characterRarity}
           />
           <CharacterSelectButton
             name="characterRarity"
             label="LR"
-            onChange={setCharacterRarity}
-            currentSelect={characterRarity}
           />
           <CharacterSelectButton
             name="characterRarity"
+            value=""
             label="ALL"
-            onChange={() => setCharacterRarity("")}
-            currentSelect={characterRarity}
+            defaultChecked
           />
         </div>
 
         <div className="order-5 bg-orange-300 rounded-md border-2 border-slate-900 flex 2xl:h-12 2xl:mt-5">
-          <button
-            className="pr-10 pl-10 pt-2 pb-2 relative hover:bg-orange-400 m-0.5"
+        <label
+          for="isUserDeck"
+        >
+          <input
+            type="checkbox"
```

Comment:
Similar to the radio, made this a checkbox that looks like a button to let the form manage the data

Checkbox is omitted from formData if not checked, otherwise gives the "value" which I set to `true` here